src/triageTemplate.ts

```diff
@@ -86,16 +88,21 @@ export const triageBugReport = async (
   const actualBehavior = components['Actual Behavior'].raw;
   if (actualBehavior === '') missingInfo.push('Actual Behavior');
 
-  if (missingInfo.length > 0) {
+  if (!update && missingInfo.length > 0) {
```

Comment:
I don't think the change on line 91 does what you intended -- you can now reach line 94 even if `missingInfo.length > 0`

Suggestion: have pass the is-an-update context along to `utils.notifyMissingInfo()` so that it can decide how to handle it. This has the advantage that the decision on how to handle the notification happens once, inside `utils.notifyMissingInfo()`, rather than once per template type